bin/release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+
+release() {
+  # Install dependencies.
+  yarn
+
+  # Bump version.
+  yarn version ${1:-patch}
+
+  # Get the new version number.
+  local version=`grep -m1 "\"version\"" package.json | awk -F: '{ print $2 }' | sed 's/[", ]//g'`
+
+  # Generate changelog.
+  node bin/cli.js --base-branch master --future-release ${version} --rebuild
```

Comment:
`master` is the default and can be omitted. Also, I don't think `--rebuild` is necessary